### DISCUSSION TITLE
What entities should I expect?

### DISCUSSION THREAD

**Discussion by pgm83** [robbrad/UKBinCollectionData#1007] [Q&A]
Just trying to set this up with South Lanarkshire Council, but all I get is an entity with Raw JSON in it's name, is that expected?

I'm assuming I've got something wrong somewhere, but not sure where!

**Comment by robbrad** [comment]
Logs are going to help here guys

**Reply by pgm83** [reply]
Mine started working a few days later with no changes from my HA side so assume something changed at the council source?

**Reply by Ukchris79** [reply]
does your URL go: 

python collect_data.py SouthLanarkshireCouncil https://www.southlanarkshire.gov.uk/directory_record/number/streetname_town

**Comment by Ukchris79** [comment]


**Reply by Ukchris79** [reply]
I have obviously redacted the last part of the address

**Comment by pgm83** [comment]
Yeh that’s right.

**Comment by pgm83** [comment]
Sorry actually in the config it’s just the url. I thought the python
section was handled in the code.

**Reply by Ukchris79** [reply]
You are a superstar, thanks.
